Explain Draft Pull Requests to communicate with others (#205)
kawalcovid19/wargabantuwarga.com: CONTRIBUTING.md

2. To start working on it, ensure that you create a new branch from the `main` branch, then commit and push your changes **as soon as possible** no matter how small they are.
3. Then **create a new pull request** while [marking it as a **draft pull request**](https://github.blog/2019-02-14-introducing-draft-pull-requests/) to signal the other contributors that [it's a work in progress](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request). This is necessary to signal the other contributors that there are work in progress for that particular issue.

### Utilising Draft Pull Requests for Communication

> Draft Pull Requests is a feature provided by GitHub as a mean to communicate with contributors. When we create a Draft Pull Request, we can't merge it until it's marked as Ready for review. This is a better approach to use it as a communication tool between contributors, and we can provide information that we are currently working on an issue.

Reviewer: ```suggestion
Draft Pull Requests is a feature provided by GitHub as a means to communicate with contributors. When we create a Draft Pull Request, we can't merge it until it's marked as Ready for review. This is a better approach to use as a communication tool between contributors, and we can provide information that we are currently working on an issue.
```

> Draft Pull Requests is a feature provided by GitHub as a mean to communicate with another developer. When we create a Draft Pull Request, we can't merge it until it's marked Ready for review. This is a better approach to use it as a communication tool between developers, and we can provide an information that we are currently are working on an issue.

Reviewer 2: ```suggestion
Draft Pull Requests is a feature provided by GitHub as a mean to communicate with contributors. When we create a Draft Pull Request, we can't merge it until it's marked as Ready for review. This is a better approach to use it as a communication tool between contributors, and we can provide information that we are currently working on an issue.
```

Reviewer 2: it might be more friendly to use word contributors instead developers because not only developer can contribute to this repo. It could be translator or documentation expert.

Author: I have spent some time thinking about the correct word for that part, thank you for the input!

Also need to change the translated version too

2. Untuk mulai mengerjakannya, pastikan **_create branch_** dari `main`, lalu segera _commit_ dan _push_ perubahannya **sesegera mungkin** sekecil apapun itu.
3. Kemudian **buat _pull request_** baru dengan [menjadikannya sebagai **_draft pull request_**](https://github.blog/2019-02-14-introducing-draft-pull-requests/) untuk memberikan sinyal kepada kontributor lainnya bahwa [_pull request_ ini masih sedang dalam pengerjaan](https://docs.github.com/en/github/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request). Dengan demikian, orang lain yang ingin mengerjakan _issue_ yang sama akan berpikir dua kali sebelum ikut mengerjakannya.

### Memanfaatkan _Draft Pull Requests_ untuk Berkomunikasi

> _Draft Pull Requests_ merupakan sebuah fitur yang disediakan oleh GitHub, yang bertujuan sebagai alat berkomunikasi bagi para kontributor. Ketika kita membuat sebuah Draft Pull Request, kita tidak bisa menggabungkannya kecuali itu sudah ditandai sebagai _Ready for review_. Ini merupakan pendekatan yang lebih baik untuk menggunakannya sebagai alat berkomunikasi antar kontributor, dan kita juga bisa memberikan informasi bahwa kita sedang mengatasi sebuah isu.

Reviewer: ```suggestion
_Draft Pull Requests_ merupakan sebuah fitur yang disediakan oleh GitHub, yang bertujuan sebagai alat berkomunikasi bagi para kontributor. Ketika kita membuat sebuah _Draft Pull Request_, kita tidak bisa menggabungkannya melainkan kita sudah menandainya sebagai _Ready for review_. Ini merupakan pendekatan yang lebih baik untuk menggunakannya sebagai alat berkomunikasi antar kontributor, dan kita juga bisa memberikan informasi bahwa kita sedang mengatasi sebuah isu.
```

Langkah-langkah untuk membuat _Draft Pull Request_:

1. _Commit_ dan _push_ perubahan terbaru ke _forked repository_ Anda.
2. Pergi ke bagian _Pull requests_ pada _forked repository_ Anda, dan klik _New pull request_.
![Petunjuk-1](https://user-images.githubusercontent.com/46013258/126284390-c2bd1aa6-fdc2-4aa6-a945-031f02db038e.png)
3. Pilih _forked repository_ Anda sebagai _head repository_, dan pilih _branch_ tempat Anda membuat perubahan untuk bagian _compare_.
![Petunjuk-2](https://user-images.githubusercontent.com/46013258/126285036-27b49325-62a2-4a6c-b216-5bae261788da.png)
4. Berikan sebuah judul dan beberapa deskripsi mengenai _pull request_ Anda, lalu pilih _Create draft pull request_ (seperti pada gambar di bawah) dan klik tombol berwarna hijau.
![Petunjuk-3](https://user-images.githubusercontent.com/46013258/126286179-04341e30-1224-49cb-9b9a-3c3aee99c308.png)

> 5. Jangan lupa untuk menandai _Draft Pull Request_ Anda sebagai _ready for _review ketika Anda sudah melakukan semua perubahan yang diinginkan.

Reviewer: ```suggestion
5. Jangan lupa untuk menandai _Draft Pull Request_ Anda sebagai _Ready for review_ ketika Anda sudah melakukan semua perubahan yang diinginkan.
```
